### PR TITLE
chore(stats): replace all `::float` by `::TEXT`

### PR DESCRIPTION
**Pull request by yjhmelody**

### CODE REVIEW

File: stats/stats/src/charts/lines/average_gas_price.rs [hunk 1]
```diff
@@ -108,8 +108,6 @@ impl StatementFromRange for AverageGasPriceStatement {
 pub type AverageGasPriceRemote =
     RemoteDatabaseSource<PullAllWithAndSort<AverageGasPriceStatement, NaiveDate, f64>>;
```
Comment (bragov4ik) [inline stats/stats/src/charts/lines/average_gas_price.rs:109]:
```suggestion
    RemoteDatabaseSource<PullAllWithAndSort<AverageGasPriceStatement, NaiveDate, String>>;
```

**Comment by bragov4ik** [comment]
I suggest to run tests locally before pushing to CI for easier development :)

**Comment by yjhmelody** [comment]
I will close this PR. I will close this PR. since the float converts in rust side is different with pg side.

```rs
---- charts::counters::average_block_time::tests::update_average_block_time stdout ----
thread 'charts::counters::average_block_time::tests::update_average_block_time' panicked at stats/src/tests/simple_test.rs:368:5:
assertion failed: `(left == right)`

Diff < left / right > :
<802200.0833333334
>802200.083333333333
```

**Comment by yjhmelody** [comment]
```
Diff < left / right > :
<802200.0833333334
>802200.083333333333
```

**Comment by bragov4ik** [comment]
you could modify tests to account for the precision errors (basically paste new values if they are very close)

**Pull request closed without merging**